site/content/posts/announcing-kudo-0.3.0.md

```diff
@@ -0,0 +1,69 @@
+
+### Package Format
+
+[KEP-10](https://github.com/kudobuilder/kudo/blob/v0.3.0/keps/0010-package-manager.md) for the package manager is underway, with release providing 2 features; package repository and a package format.
+
+The general format allows for a number of files, but starts with an `operator.yaml`.  A common layout might be:
+
+```
+operator.yaml
+params.yaml
+templates/
+     <additional files>.yaml
+```
+The `operator.yaml` defines the operator and its plans.  `params.yaml` defines the params to be used through all templated files.  While the templated files are defined under the templates folder. The templated files are referenced in the `operator.yaml` file.  The zookeeper operator provides a [good example](https://github.com/kudobuilder/operators/tree/a1f4cf/repository/zookeeper/0.1.0).
+
+
+These package files are group together in a folder or subfolder or can be tarred together in a `tgz` file providing a couple of ways to install an operator.
+
+### Package Installation Options
+
+Introduced with this released are 2 ways to install an operator to KUDO. The first is designed for operator developers which is to install from the local filesystem. This can be either a folder with the operator package files or a tarball of those files.
+
+`kubectl kudo install ./kafka` or `kubectl kudo install kafka.tgz`
+
+This release also introduces a repository which allows for automatic discovery of operators to be installed.
+`kubectl kudo install kakfa`
+
+**Note:**  The order of operation when installing an operator is to check the local filesystem first, then to query the repository.   
+
+### Skip Instance
+
+kudoctl install now supports `-skip-instance` flag which skips installing an Instance when installing Operator and OperatorVersions. This is useful in a testing scenario - we install all of the Operators and OperatorVersions but don't want to install the Instances until later.
+
+### Test Harness
+The test harness outlined in [KEP-8](https://github.com/kudobuilder/kudo/blob/v0.3.0/keps/0008-operator-testing.md) has been implemented.  The harness described in [testing.md](https://github.com/kudobuilder/kudo/blob/v0.3.0/docs/testing.md) provides a mechanism for operator developers to create end to end integration testing for their frameworks.  Introduced with the test harness is the `kind: TestAssert` to assert operator state.  [Example tests](https://github.com/kudobuilder/operators/tree/v0.3.0/repository/zookeeper/tests/zookeeper-upgrade-test) are provided for the zookeeper operator.
```

Comment:
Maybe change the link to https://kudo.dev/docs/testing

Reply:
that url is above for the word harness that could definitely be improved.

Reply:
i included this in my suggested change, so it should be updated already